packages/apib-parser/lib/adapter.js

```diff
@@ -42,9 +42,29 @@ function parse({
     requireBlueprintName,
   };
 
-  return drafter.parse(source, options);
+  return drafter.parse(source, options).then((result) => {
+    const parseResult = namespace.fromRefract(result);
+    const isAnnotation = element => element.element === 'annotation';
+    const { Link } = namespace.elements;
+
+    if (!isAnnotation(parseResult.content[0])) {
+      const link = new Link();
+
+      link.title = 'Apiary Blueprint';
```

Comment:
```suggestion
      link.title = 'API Blueprint';
```

This is the API Blueprint adapter

Reply:
changed